tool/tsh/proxy.go

```diff
@@ -351,3 +383,11 @@ Use following credentials to connect to the {{.database}} proxy:
   cert_file={{.cert}}
   key_file={{.key}}
 `))
+
+// dbProxyAuthTpl is the message that's printed for an authenticated db proxy.
+var dbProxyAuthTpl = template.Must(template.New("").Parse(
+	`Started authenticated local proxy for the database "{{.database}}" on {{.address}}.
+
+Use the following command to connect to the database:
+  $ {{.command}}
```

Comment:
The "mtls" mode should also work for GUIs? Not suggesting any change here. Just curious if this should be default mode eventually.

Reply:
Yep. I was planning to look into updating the docs after this merges.